cli/br/brooklyn.go

```diff
@@ -30,27 +30,14 @@ import (
 	"path/filepath"
 )
 
-func getNetworkCredentialsFromConfig(yamlMap map[string]interface{}) (string, string, string, bool) {
-	var target, username, password string
-	var skipSslChecks bool
-	target, found := yamlMap["target"].(string)
-	if found {
-		auth, found := yamlMap["auth"].(map[string]interface{})
-		if found {
-			creds := auth[target].(map[string]interface{})
-			username, found = creds["username"].(string)
-			if found {
-				password, found = creds["password"].(string)
-			}
-		}
-		skipSslChecks, _ = yamlMap["skipSslChecks"].(bool)
-	}
-	return target, username, password, skipSslChecks
-}
-
 func main() {
 	config := io.GetConfig()
-	target, username, password, skipSslChecks := getNetworkCredentialsFromConfig(config.Map)
+	skipSslChecks := config.GetSkipSslChecks()
+	target, username, password, err := config.GetNetworkCredentials()
+	if err != nil && !isLogin(os.Args) {
```

Comment:
It might read nicer to skip config.GetNetworkCredentials() if it is login, rather than ignore the error